rhel/SPECS/pmm-server.spec

```diff
@@ -63,13 +61,14 @@ rm -rf %{pmm_repo}-%{pmm_commit}
 %doc README.md CHANGELOG.md
 %{_sysconfdir}/supervisord.d
 %{_sysconfdir}/alertmanager.yml
-%{_sysconfdir}/nginx/conf.d/pmm.conf
-%{_sysconfdir}/nginx/conf.d/pmm-ssl.conf
 %{_datadir}/percona-dashboards/import-dashboards.py*
 %{_datadir}/%{name}
 
 
 %changelog
+* Tue Jun 30 2021 Nikita Beletskii <nikita.beletskii@percona.com> - 2.20.0-2
```

Comment:
This date would lead to build error

Reply:
Do you mean the day of the week? It just has format check, but correctness but it's a mistake anyway. Fixed!

Reply:
I mean, in the past the builds used to fail, since the builder checks the dates for correctness.